### DISCUSSION TITLE
Make the tabs better visible

### DISCUSSION THREAD

**Discussion by muellerto** [doublecmd/doublecmd#1873] [Suggestions]
I have a lot of tabs. It's hard to recognize which one is active. In dark mode it's a bit better but in light mode it looks like this:

![3](https://github.com/user-attachments/assets/ef19d2e8-a456-436b-a646-a97523652972)

You can't even say how many tabs are there? And which is the active one? Yes, I know, you can see this all. But you must have very sharp eyes.

**Comment by alexx2000** [comment]
It is a standard Windows tab-control. So it has a system look. It is a Windows 10/11 bad flat style.
DC marks an active tab by blue-line.

**Reply by muellerto** [reply]
Yes, sure. But you can owner-draw this control for sure. I suggest to use a full background color for the active tab and better dividers.

OK, perhaps the user could use [ and ] in the tab names as a workaround.

**Comment by muellerto** [comment]
I added a vertical bar and a space at the beginning of the right tabs. Looks not bad for my taste:

![4](https://github.com/user-attachments/assets/9755e33a-e1f1-4b3c-8752-45149a47aea7)

The bar is the Unicode character 0x2502. The space left from the vertical bar comes from the tab control.
